odl/util/utility.py

```diff
@@ -805,6 +805,43 @@ def pkg_supports(feature, pkg_version, pkg_feat_dict):
     return False
 
 
+class NumpyRandomSeed(object):
+    """Context manager for numpy randoms seeds."""
+
+    def __init__(self, seed):
+        """
+
+        Parameters
+        ----------
+        seed : int or None
+            Seed value for the random number generator.
+            ``None`` is interpreted as keeping the current seed.
+
+        Examples
+        --------
+        Always returns same value with same seed:
```

Comment:
Make text a bit nicer, e.g., "Use this to make drawing pseudo-random numbers repeatable:"